osbs/build/plugins_configuration.py

```diff
@@ -531,6 +531,39 @@ def render_koji_delegate(self):
                 self.pt.set_plugin_arg(phase, plugin, 'triggered_after_koji_task',
                                        self.user_params.triggered_after_koji_task.value)
 
+    def render_tag_and_push(self):
+        """Configure tag_and_push plugin"""
+        phase = 'postbuild_plugins'
+        plugin = 'tag_and_push'
+
+        if self.pt.has_plugin_conf(phase, plugin):
+            if self.user_params.koji_target.value:
+                self.pt.set_plugin_arg(
+                    phase, plugin,
+                    'koji_target',
+                    self.user_params.koji_target.value
+                )
+
+    def render_fetch_sources(self):
+        """Configure fetch_sources"""
+        phase = 'prebuild_plugins'
+        plugin = 'fetch_sources'
+
+        if self.pt.has_plugin_conf(phase, plugin):
+            # required option
+            self.pt.set_plugin_arg(
+                phase, plugin,
+                'koji_build_nvr',
```

Comment:
you can also set koji_build_id from sources_for_koji_build_id

Reply:
Added